### ISSUE TITLE
Graphing mode: Attempting to invoke keypad buttons with Narrator does not work

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#956]
**Describe the bug**
With Narrator turned on, attempting to invoke buttons on the graphing mode keypad via Spacebar instead types a space into an equation.

**Steps To Reproduce**

1. Go to Graphing Mode.
2. Turn on Narrator.
3. Use Tab to navigate to the equations
4. Use Capslock + right/left arrow keys to navigate to the keypad.
Note: Navigating to keypad via Tab is blocked by Issue #948 
5. With a button highlighted, attempt to invoke the button via spacebar.
6. Observe the value is not entered; instead, a space is added to an equation.

**Expected behavior**
Users should be able to enter values from the keypad by invoking buttons via the keyboard.

**Screenshots**

**Device and Application Information**
 - OS Build: 10.0.19547.0
- Architecture: x64
- Application Version: 10.1912.4.0
- Region: en-US
- Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
Thanks for the detailed bug report! This is _by design_ in Graphing Mode.

Unlike other calculator modes where the onscreen keyboard is locked to one display input/output area, Graphing Mode supports free-form input. While we still have the on-screen keyboard, it now behaves more like a SIP and is not intended to receive keyboard or screen reader focus. _All_ operations available in the SIP can be entered with standard keyboard input. The SIP is still available for simpler pointer input.

Free-from input in other modes is tracked by #620.
